packages/app/src/cli/services/app-logs/services/output-functions.ts

```diff
@@ -0,0 +1,77 @@
+import {AppLogsOnFunctionRunCallback, AppLogsOnErrorCallback} from '../types.js'
```

Comment:
This file is for the custom logic that `pollAppLogs` needs to make it more re-useable between the `logs` and `dev` command. It defines a `onFunctionRunCallback` and `onErrorCallback` that are later used in `pollAppLogs` by either dev or logs.